clients/ordering/src/components/checkout.tsx

```diff
@@ -0,0 +1,266 @@
+              </ListItem>
+              <ListItem>
+                <ListItemText primary={'Tax'} />
+                <Typography variant={'subtitle2'}>
+                  {'$' + (this.props.finalTotal * 0.09).toFixed(2)}
+                </Typography>
+              </ListItem>
+              <ListItem>
+                <ListItemText primary={'Tip'} />
+                <Slider
+                  className={classes.slider}
+                  value={value}
+                  min={0}
+                  max={100}
+                  step={1}
+                  onChange={this.handleTipChange}
+                />
+                <ListItemText
+                  primary={value + '%'}
+                  style={{ marginRight: 300 }}
+                />
+                <Typography variant={'subtitle2'}>
+                  {'$' + ((value / 100) * this.props.finalTotal).toFixed(2)}
+                </Typography>
+              </ListItem>
+              <ListItem>
+                <ListItemText primary={'Total'} />
+                <Typography variant={'subtitle2'} className={classes.totalText}>
+                  {'$' +
+                    (
+                      (value / 100) * this.props.finalTotal +
+                      this.props.finalTotal +
+                      this.props.finalTotal * 0.09
+                    ).toFixed(2)}
+                </Typography>
+              </ListItem>
+            </List>
+            <Typography variant={'h6'} gutterBottom={true} align={'left'}>
+              Billing Address
+            </Typography>
+            <Grid container={true} spacing={24}>
+              <Grid item={true} xs={12} sm={6}>
+                <TextField
+                  required={true}
+                  id={'firstName'}
+                  name={'firstName'}
+                  label={'First name'}
+                  fullWidth={true}
+                  autoComplete={'fname'}
+                />
+              </Grid>
+              <Grid item={true} xs={12} sm={6}>
+                <TextField
+                  required={true}
+                  id={'lastName'}
+                  name={'lastName'}
+                  label={'Last name'}
+                  fullWidth={true}
+                  autoComplete={'lname'}
+                />
+              </Grid>
+              <Grid item={true} xs={12}>
+                <TextField
+                  required={true}
+                  id={'address1'}
+                  name={'address1'}
+                  label={'Address line 1'}
+                  fullWidth={true}
+                  autoComplete={'biling address-line1'}
+                />
+              </Grid>
+              <Grid item={true} xs={12}>
+                <TextField
+                  required={true}
+                  id={'address2'}
+                  name={'address2'}
+                  label={'Address line 2'}
+                  fullWidth={true}
+                  autoComplete={'biling address-line2'}
```

Comment:
Typo?

Reply:
Where?

Reply:
`biling` instead of `billing`

Reply:
Ah, ok. I changed it but it didn't seem to matter.